libs/cms-api/src/index.ts

```diff
@@ -214,8 +214,7 @@ async function getTelegramSubscriptionsForAccounts({
 
   if (error) {
     console.error(
-      `Error ${response.status} getting telegram subscriptions: ${response.url}. Page${page}`,
-      error
+      `Error ${response.status} getting telegram subscriptions: ${response.url}. Page${page}`
```

Comment:
Sure we don't want to log the error with the stack trace?

Reply:
yes, cause we retrow